include/zephyr/xen/public/grant_table.h

```diff
@@ -126,7 +126,7 @@ typedef uint32_t grant_ref_t;
  * Version 1 of the grant table entry structure is maintained purely
  * for backwards compatibility.  New guests should use version 2.
  */
-#if __XEN_INTERFACE_VERSION__ < 0x0003020a
+#if CONFIG_XEN_INTERFACE_VERSION < 0x0003020a
```

Comment:
Maybe it is easier to add `#define __XEN_INTERFACE_VERSION__ CONFIG_XEN_INTERFACE_VERSION` somewhere?

Reply:
It seems, the same approach for XEN_DOMCTL_INTERFACE_VERSION also could be right thing to do?